pkg/localstore/reserve.go

```diff
@@ -105,22 +73,91 @@ func (db *DB) UnreserveBatch(id []byte, radius uint8) (evicted uint64, err error
 		return 0, err
 	}
 
-	if reserveSizeChange > 0 {
-		batch = new(leveldb.Batch)
-		if err := db.incReserveSizeInBatch(batch, -int64(reserveSizeChange)); err != nil {
+	// trigger garbage collection if we reached the capacity
+	if gcSize >= db.cacheCapacity {
+		db.triggerGarbageCollection()
+	}
+
+	return reserveSizeChange, nil
+}
+
+var unpinBatchSize = 10000
```

Comment:
Can be a constant.

Reply:
So in the test, I am changing this value. It was too large for the test.

Reply:
Sorry, didn't catch that.